towntest_chest/init.lua

```diff
@@ -404,6 +404,16 @@ minetest.register_abm({
 	action = towntest_chest.build,
 })
 
+
+minetest.register_craft({
+        output = 'towntest_chest:chest',
+        recipe = {
+                {'default:mese_crystal', 'default:chest_locked', 'default:mese_crystal'},
+                {'default:book', 'default:diamond',    'default:book'},
```

Comment:
Maybe I'm hard even the life using nano as editor in VT. Copy&Paste converts the tabs to spaces in console.  I did some whitespace fixing now.
Explaing the recipe? My thoughts were the chest is precious so mese and diamond should be in recipe. books for the knowledge about buildings and locked chests ;) If you have a better idea feel free to change the recipe.